.github/workflows/main.yaml

```diff
@@ -7,14 +7,15 @@ jobs:
     strategy:
       matrix:
         platform: [ubuntu-latest]
-        node: [ '*' ]
+        node: ['*']
     name: CI - Node latest (${{ matrix.platform }})
     runs-on: ${{ matrix.platform }}
     steps:
-      - uses: actions/checkout@v1
-      - uses: actions/setup-node@v1
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
         with:
           node-version: ${{ matrix.node }}
+        - run: npm i -g npm
```

Comment:
This is probably not correctly intended. See the error at the bottom of https://github.com/terser/html-minifier-terser/actions/runs/1007155491

Reply:
Yeah. I had it initially when the package-lock.json was committed. While reverting it I forgot to remove it.

Reply:
done.